dagshub/common/config.py

```diff
@@ -0,0 +1,14 @@
+import appdirs
+import os
+
+HOST_KEY = "DAGSHUB_CLIENT_HOST"
+DEFAULT_HOST = "https://dagshub.com"
+CLIENT_ID_KEY = "DAGSHUB_CLIENT_ID"
+DEFAULT_CLIENT_ID = "32b60ba385aa7cecf24046d8195a71c07dd345d9657977863b52e7748e0f0f28"
+CACHE_LOCATION_KEY = "DAGSHUB_CLIENT_CONFIG"
+DEFAULT_CACHE_LOCATION = os.path.join(appdirs.user_cache_dir("dagshub"), "tokens")
+CONFIG_SCHEMA_VERSION = "1"
```

Comment:
Clarify `config` vs `tokens` terminology to me. If we decide to actually save configs for the client, are we going to use the `user_cache_dir` as well? 
If we introduce a new `"dagshub-config"` file, does the schema change? Or does it refer to the `"tokens"` schema?